tests/testthat/_snaps/rust-siterp.md

```diff
@@ -1,13 +0,0 @@
-# `cargo` or `rustup` are not found
```

Comment:
Why this snapshot is now removed?

Reply:
Please pay attention to the filename. This file seems to have been added by typo and is not really used.

Reply:
Oh, ok, I see, I now remember I messed up with test naming when I was developing `rust_sitrep()` and forgot to remove the snapshot!